pkgs/development/compilers/llvm/16/compiler-rt/freebsd-i386.patch

```diff
@@ -0,0 +1,21 @@
+--- a/lib/builtins/fp_lib.h	1969-12-31 16:00:01.000000000 -0800
```

Comment:
Apologies if I'm missing something obvious: do these two patch files need to be specified in `llvm/16/default.nix`?

Reply:
Yes... they do need to be specified. It seems they haven't been, which means they're not strictly required for the native FreeBSD stdenv, but will be required for other packages at some point. I'll fix this...